cmd/buildah/version.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+
+	ispecs "github.com/opencontainers/image-spec/specs-go"
+	rspecs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/projectatomic/buildah"
+	"github.com/urfave/cli"
+)
+
+//Overwritten at build time
+var (
+	gitCommit string
+	buildInfo string
+)
+
+//Function to get and print info for version command
+func versionCmd() {
+
+	//converting unix time from string to int64
+	buildTime, err := strconv.ParseInt(buildInfo, 10, 64)
+	if err != nil {
+		fmt.Println("Error getting build time:", err, buildTime)
+	}
+
+	fmt.Println("Version:	", buildah.Version)
+	fmt.Println("Go Version:	", runtime.Version())
+	fmt.Println("Image Spec:	", ispecs.Version)
+	fmt.Println("Runtime Spec:	", rspecs.Version)
+	fmt.Println("Git Commit:	", gitCommit)
+
+	//Prints out the build time in readable format
+	fmt.Println("Built:		", time.Unix(buildTime, 0).Format(time.ANSIC))
+	fmt.Println("OS/Arch:	", runtime.GOOS+"/"+runtime.GOARCH)
+}
+
+//cli command to print out the full version of kpod
+var versionCommand = cli.Command{
+	Name:  "version",
+	Usage: "Display the Buildah Version Information",
+	Action: func(context *cli.Context) error {
+		versionCmd()
+		return nil
+	},
```

Comment:
No need for this wrapper if `versionCmd()` is changed to return an error -- either one received from `strconv.ParseInt()` or `nil`, if `versionCmd()` runs without problems.